base/applications/rapps/settings.cpp

```diff
@@ -165,27 +163,18 @@ LoadAllSettings(CRegKey &key, SETTINGS_INFO &settings)
     while (InfoListPosition)
     {
         SettingsField *Info = infoFields.GetNext(InfoListPosition);
-        if (!Info->Load(key))
-        {
-            bAllSuccess = FALSE;
-            // TODO: error log
-        }
+        if (Info->Load(key))
+            bLoadedAny = TRUE;
+        //else
```

Comment:
```suggestion
        bLoadedAny = Info->Load(key);
        // if (!bLoadedAny)
```

Reply:
No, it would have to be `bLoadedAny |= Info->Load(key);` if you want to do it your way but that makes the `else` more complicated.